slides/kube/kubectlget.md

```diff
@@ -212,6 +212,8 @@ The error that we see is expected: the Kubernetes API requires authentication.
 
 *Ding ding ding ding ding!*
 
+`kube-system`: "The namespace for objects created by the Kubernetes system"
```

Comment:
After reading that again — I'm thinking … "Wait, all objects are created by the Kubernetes system, aren't they?"

What about, "The `kube-system` namespace holds the objects that form Kubernetes' control plane: its API server, controllers … It provides a clean way to separate them logically from our other objects." Hmmm. WDYT?

Reply:
Yes. Yes, they are. Hopefully this brief version works!